components/vf-componenet-rollup/index.scss

```diff
@@ -1,3 +1,4 @@
+@import 'package.variables.scss';
 /*
```

Comment:
when we start to use the `.min.css` this needs to be written as `/!*` for Sass to not remove it on compiling. AFAIK.

Actually this may be negated by gulp settings

Reply:
Ah, ok follow now. 

We do need the loud comments, pushing a change shortly to:

![image](https://user-images.githubusercontent.com/928100/63006072-10e34f80-be7e-11e9-881e-528c66facf44.png)